rom-elks.c

```diff
@@ -141,7 +140,7 @@ static int int_10h ()
 		notimp:
 			fflush(stdout);
 			printf ("fatal: INT 10h: AH=%hxh not implemented\n", ah);
-			assert (0);
```

Comment:
Please replace 'fatal' by 'error' as the emulator does not exit anymore on non-implemented BIOS function.

Reply:
Done